docs/api/README.md

```diff
@@ -254,6 +254,27 @@ metrics. By default, the Service exposes metrics on port `:9090`. For more
 details on the structure of the metrics, see
 <https://github.com/grpc-ecosystem/go-grpc-prometheus#metrics>.
 
+## Health
+
+The API Server includes gRPC and REST endpoints for monitoring serving status of the API server as well as serving 
+status of individual services.
+
+### Checking Status
+
+````sh
+# Check status of API server using gRPC
+grpcurl --insecure localhost:8080 grpc.health.v1.Health/Check
```

Comment:
This the the first time we mention the `grpcurl` tool, maybe add it here https://github.com/tektoncd/results/blob/main/docs/DEVELOPMENT.md?plain=1#L17